src/pages/EmployeeListing/components/EmployeeCard/EmployeeCard.tsx

```diff
@@ -29,7 +23,10 @@ export function EmployeeCard({ employee }: { employee: Employee }) {
           }}
         />
       </div>
-      <img src={profileUrl} alt="employee profile" />
+      <img
+        src={employee.profilePic ?? defaultprofileImg}
+        alt="employee profile"
+      />
       <div>
         <p className="view-emp-id">#{employee.employeeId}</p>
```

Comment:
emp-id

Reply:
should i change the class-name from view-emp-id to just emp-id ?